core/src/test/scala/integration/kafka/api/BaseConsumerTest.scala

```diff
@@ -111,6 +115,40 @@ abstract class BaseConsumerTest extends AbstractConsumerTest {
 }
 
 object BaseConsumerTest {
+  // We want to test the following combinations:
+  // * ZooKeeper and the generic group protocol
+  // * KRaft and the generic group protocol
+  // * KRaft with the new group coordinator enabled and the consumer group protocol
+  def getTestQuorumAndGroupProtocolParametersAll() : java.util.stream.Stream[Arguments] = {
+    java.util.stream.Stream.of(
+        Arguments.of("zk", "generic"),
+        Arguments.of("kraft", "generic"),
+        Arguments.of("kraft+kip848", "consumer"))
```

Comment:
We also need to test the `generic` with `kraft+kip848`. This is what all the tests with `kraft+kip848` prior to your change did.